### DISCUSSION TITLE
Display several index time series together

### DISCUSSION THREAD

**Discussion by tgexport** [highfestiva/finplot#511] [General]
Hi,
I want to show 2 different timeframes together in the chart, but their index and candle width are different, and in the situation where the smaller time index ends, it shows the larger time with the smaller time index.
Can this be done?
For example, here the time is 1 minute and 5 minutes. As long as we have a 1-minute candle, by setting the candle_width=5 of the 5-minute candle, the 5-minute candle time will show correctly, but when the 1-minute candle time ends, the 5-minute time candles will not show correctly.
I don't want to create a 5-minute candle from a 1-minute candle, the data for both is different.

![finplot-](https://github.com/highfestiva/finplot/assets/73376421/d04a95bf-d0ae-409b-b35b-494f57289545)

**Comment by highfestiva** [comment]
Usually these things come down to some data manipulation before calling finplot, but I don't understand what you're trying to achieve and what the problem is. Either you need to supply some example I can run, or you can try extending the 5m data so it "covers" the 1m data.

**Reply by tgexport** [reply]
`
#!/usr/bin/env python3

from datetime import date, timedelta
import finplot as fplt
import yfinance as yf

now = date.today()
start_day = now - timedelta(days=6)
df = yf.download('GOOG', start_day.isoformat(), now.isoformat(), interval='5m')

start_day2 = now - timedelta(days=3)
df2 = yf.download('GOOG', start_day2.isoformat(), now.isoformat(), interval='1m')

ax = fplt.create_plot('Alphabet Inc.', rows=1)

fplt.candle_bull_color = '#000'
fplt.candle_bear_color = '#000'
fplt.candle_bull_body_color = '#fff'
fplt.candle_bear_body_color = '#fff'
plot_5m = fplt.candlestick_ochl(df[['Open','Close','High','Low']],candle_width=5)
plot_5m.x_offset = 1.4

fplt.candle_bull_color = 'blue'
fplt.candle_bear_color = 'blue'
plot_1m = fplt.candlestick_ochl(df2[['Open','Close','High','Low']])

fplt.show()

`


Run this code, you will understand what I mean.
Up to half of the chart, both timeframes are shown correctly, but further, the candles of the higher timeframe is not shown correctly.

**Comment by tgexport** [comment]
![finplot2](https://github.com/highfestiva/finplot/assets/73376421/c1bbc510-4b44-4769-a049-c53e76a66795)

**Reply by highfestiva** [reply]
The candles are shown equidistant, so if you want the same x-resolution all the way, you need to supply both 5m and 1m data all the way.
